pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1141,7 +1160,19 @@ func TestPriorityQueue_Update(t *testing.T) {
 				if tt.wantAddedToNominated && len(q.nominator.nominatedPods) != 1 {
 					t.Errorf("Expected one item in nomindatePods map: %v", q.nominator)
 				}
+				if tt.wantQ == notInAnyQueue {
+					if _, exists, _ := q.podBackoffQ.Get(newQueuedPodInfoForLookup(newPod)); exists {
```

Comment:
We checked `exists` for each queue before in L1135, L1142, L1149 already, right? Do we really need to check again here?

Maybe skip for podInfo check in L1156 if `wantQ == notInAnyQueue` is enough?